response_types.go

```diff
@@ -9,27 +9,30 @@ import (
 	"time"
 )
 
-type accessToken struct {
+type AccessToken struct {
 	CreatedAt   int64
 	ExpiresIn   int64  `json:"expires_in,string"`
 	AccessToken string `json:"access_token"`
 	TokenType   string `json:"token_type"`
 }
 
-func (token *accessToken) isValid() bool {
-	createdAt := token.CreatedAt
-	expiresIn := token.ExpiresIn
-
-	expirationLimit := createdAt + expiresIn
+func (token *AccessToken) isValid() bool {
```

Comment:
Validness of the token could include other things that are not the goal of this method, so maybe we should rename this to `isExpired`, since expiration is the only parameter this method verifies.

Reply:
+1